IlcStandaloneGCSupport Not Working For NativeAoT

### Description

I am getting the following error while trying to compile a Native AoT Application after specifying IlcStandaloneGCSupport to be true.

### Reproduction Steps

In the csproj of the Native AoT project, I am setting: 

<IlcStandaloneGCSupport>true</IlcStandaloneGCSupport>

### Expected behavior

This behavior should compile. 

### Actual behavior

Fails with the following callstack: 

musharm:BasicMinimalApi:% ~/net9/dotnet publish -c Release                                        <main ✗>
MSBuild version 17.9.0-preview-23557-02+5d1509792 for .NET
  Determining projects to restore...
  Restored /home/mrsharm/Benchmarks/src/BenchmarksApps/BasicMinimalApi/BasicMinimalApi.csproj (in 174 ms).
/home/mrsharm/net9/sdk/9.0.100-alpha.1.23560.7/Sdks/Microsoft.NET.Sdk/targets/Microsoft.NET.RuntimeIdentifierInference.targets(311,5): message NETSDK1057: You are using a preview version of .NET. See: https://aka.ms/dotnet-support-policy [/home/mrsharm/Benchmarks/src/BenchmarksApps/BasicMinimalApi/BasicMinimalApi.csproj]
  BasicMinimalApi -> /home/mrsharm/Benchmarks/src/BenchmarksApps/BasicMinimalApi/bin/Release/net9.0/linux-x64/BasicMinimalApi.dll
  Generating native code
  /usr/bin/ld.bfd: /usr/bin/ld.bfd: DWARF error: invalid or unhandled FORM value: 0x25
  **/home/mrsharm/.nuget/packages/runtime.linux-x64.microsoft.dotnet.ilcompiler/9.0.0-alpha.1.23558.17/sdk/libstandalonegc-enabled.a(clrgc.enabled.cpp.o): in function `standalone::GCToEEInterface::LogStressMsg(unsigned int, unsigned int, StressLogMsg const&)':
  clrgc.enabled.cpp:(.text._ZN10standalone15GCToEEInterface12LogStressMsgEjjRK12StressLogMsg[_ZN10standalone15GCToEEInterface12LogStressMsgEjjRK12StressLogMsg]+0xd): undefined reference to `GCToEEInterface::LogStressMsg(unsigned int, unsigned int, StressLogMsg const&)'**
collect2 : error : ld returned 1 exit status [/home/mrsharm/Benchmarks/src/BenchmarksApps/BasicMinimalApi/BasicMinimalApi.csproj]
/home/mrsharm/.nuget/packages/microsoft.dotnet.ilcompiler/9.0.0-alpha.1.23558.17/build/Microsoft.NETCore.Native.targets(363,5): error MSB3073: The command ""gcc" "obj/Release/net9.0/linux-x64/native/BasicMinimalApi.o" -o "bin/Release/net9.0/linux-x64/native/BasicMinimalApi" -gz=zlib -fuse-ld=bfd /home/mrsharm/.nuget/packages/runtime.linux-x64.microsoft.dotnet.ilcompiler/9.0.0-alpha.1.23558.17/sdk/libbootstrapper.o /home/mrsharm/.nuget/packages/runtime.linux-x64.microsoft.dotnet.ilcompiler/9.0.0-alpha.1.23558.17/sdk/libRuntime.ServerGC.a /home/mrsharm/.nuget/packages/runtime.linux-x64.microsoft.dotnet.ilcompiler/9.0.0-alpha.1.23558.17/sdk/libeventpipe-enabled.a /home/mrsharm/.nuget/packages/runtime.linux-x64.microsoft.dotnet.ilcompiler/9.0.0-alpha.1.23558.17/sdk/libstandalonegc-enabled.a /home/mrsharm/.nuget/packages/runtime.linux-x64.microsoft.dotnet.ilcompiler/9.0.0-alpha.1.23558.17/sdk/libstdc++compat.a /home/mrsharm/.nuget/packages/runtime.linux-x64.microsoft.dotnet.ilcompiler/9.0.0-alpha.1.23558.17/framework/libSystem.Native.a /home/mrsharm/.nuget/packages/runtime.linux-x64.microsoft.dotnet.ilcompiler/9.0.0-alpha.1.23558.17/framework/libSystem.IO.Compression.Native.a /home/mrsharm/.nuget/packages/runtime.linux-x64.microsoft.dotnet.ilcompiler/9.0.0-alpha.1.23558.17/framework/libSystem.Net.Security.Native.a /home/mrsharm/.nuget/packages/runtime.linux-x64.microsoft.dotnet.ilcompiler/9.0.0-alpha.1.23558.17/framework/libSystem.Security.Cryptography.Native.OpenSsl.a -g -Wl,-rpath,'$ORIGIN' -Wl,--build-id=sha1 -Wl,--as-needed -pthread -ldl -lz -lrt -lm -pie -Wl,-pie -Wl,-z,relro -Wl,-z,now -Wl,--eh-frame-hdr -Wl,--discard-all -Wl,--gc-sections" exited with code 1. [/home/mrsharm/Benchmarks/src/BenchmarksApps/BasicMinimalApi/BasicMinimalApi.csproj]

### Regression?

_No response_

### Known Workarounds

_No response_

### Configuration

_No response_

### Other information

_No response_

--- a/src/coreclr/nativeaot/Runtime/gcrhenv.cpp
+++ b/src/coreclr/nativeaot/Runtime/gcrhenv.cpp
@@ -1114,6 +1114,11 @@ void GCToEEInterface::UpdateGCEventStatus(int currentPublicLevel, int currentPub
     // TODO: Linux LTTng
 }
 
+void GCToEEInterface::LogStressMsg(unsigned level, unsigned facility, const StressLogMsg& msg)
+{
+    // TODO: Implementation
+}
+
 uint32_t GCToEEInterface::GetCurrentProcessCpuCount()
 {
     return PalGetProcessCpuCount();

